ADR/adr-0010-namespace-metadata

```diff
@@ -25,6 +25,14 @@ We will apply the following labels to Stone Soup namespaces, to make them easier
 - `stonesoup.redhat.com/namespacetype: "user-workspace-data"` for User workspaces where Applications, Components, and so on are defined
 - `stonesoup.redhat.com/namespacetype: "user-deployments"` for the namespaces where GitOps deploys applications for users
 - `stonesoup.redhat.com/namespacetype: "user-builds"` for the namespaces where the Pipeline Service has PipelineRuns
+- `stonesoup.redhat.com/workspace_id: "b2dece58-a260-11ed-a8fc-0242ac120002"` a unique identifier for the instance of the service (user or workspace UUID)
+- `stonesoup.redhat.com/external_organization: "redhat"` the Red Hat orgId
+- `stonesoup.redhat.com/resource_name: "Stonesoup"` identifier for the type of product (allows tracking multiple variants in the same metric)
+- `stonesoup.redhat.com/account_support: "standard"` Standard, Premium, etc. Must match the value from the SKU
+- `stonesoup.redhat.com/env_usage: "prod"` Development/Test, Production, etc. Must match the value from the SKU
+- `stonesoup.redhat.com/billing_model: "marketplace"` must be set to marketplace to indicate this service instance is billed through marketplace. This allows you to mark some instances as billed via marketplace (and some as not billed through marketplace)
+- `stonesoup.redhat.com/billing_marketplace: "aws"` which marketplace is used for billing
+- `stonesoup.redhat.com/billing_marketplace_account: 123456789012` the customer account identifier (numeric AWS identifier). Necessary because a customer can have more than one AWS account.
```

Comment:
What would we set these 3 values to for the MVP, when no billing setup exists yet?

Reply:
Can we omit them for MVP?